packages/schema/src/schema.graphql

```diff
@@ -173,9 +178,10 @@ type Query {
 
   allPages: [Page] @gatsbyNodes(type: "Page")
   websiteSettings: WebsiteSettings
-  viewPage(id: ID!, rid: ID, locale: String!): Page
-    @fetchEntity(type: "node", id: "$id", rid: "$rid", language: "$locale")
-    @gatsbyNode(type: "Page", id: "$id")
+  viewPage(path: String!): Page
+    @route(path: "$path")
+    @fetchEntity
+    @gatsbyPageByPath(path: "$path")
```

Comment:
Exactly. Every system uses the directives it knows. This might be confusing at some point, but i have no better idea (yet).

Reply:
Technically one directive can be implemented in multiple systems 🤔  I have to think about that.

Reply:
@Leksat: I pushed another experimental commit. Instead of re-using the `@route` and `@fetchEntity` directive, i moved that code into a custom `@pageByPath` directive, that works both in Drupal and Gatsby.

I'm not sure if thats preferable or not:

**Pro:**
* its easy to understand where things are implemented
* its easy to modify the implementation

**Con:**
* it duplicates code from the two directives

Reply:
**Semi-pro:** its a nice example of a directive implemented in multiple systems